src/utils/requests.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from "@angular/core";
+import { Headers, Http } from "@angular/http";
+
+@Injectable()
+export class Requests {
+
+	private headers = new Headers({'Content-Type': 'application/json'});
+	
+	constructor(private http: Http) { }
```

Comment:
Looks like the only way to do that is using @Inject. Can we merge it for now, and later experiment more on it?